router/event.go

```diff
@@ -63,36 +54,20 @@ func (router *Router) eventFromRequest(r *http.Request) (*eventpkg.Event, string
 			return nil, "", err
 		}
 	}
-
-	event := eventpkg.New(eventType, mime, body)
-
-	// Because event.Data is []bytes here, it will be base64 encoded by default when being sent to remote function,
-	// which is why we change the event.Data type to "string" for forms, so that, it is left intact.
-	if len(body) > 0 {
-		switch {
-		case mime == mimeJSON:
-			err := json.Unmarshal(body, &event.Data)
-			if err != nil {
-				return nil, "", errors.New("malformed JSON body")
-			}
-		case strings.HasPrefix(mime, mimeFormMultipart), mime == mimeFormURLEncoded:
-			event.Data = string(body)
-		}
+	var sourceURL string
+	if r.TLS != nil {
```

Comment:
After thinking more about this one I think it won't work in all cases.

Reply:
Why not?